### DISCUSSION TITLE
Plist? Passwords?

### DISCUSSION THREAD

**Discussion by magnusviri** [ElektraInitiative/libelektra#3661] [General]
I can't find any statements in the documentation about plist support or storing secrets like passwords. I realize this isn't a password manager but do you have any intention of working with them to retrieve secrets?

**Comment by kodebach** [comment]
I don't think we currently have a plugin that supports plist files.

For password and secret storage, you may take a look at the `crypto`, `fcrpyt` and `gpgme` plugin written by @petermax2.

**Comment by markus2330** [comment]
@magnusviri  Thank you for writing us! Yes, unfortunately we currently do not have direct support for plist, only for XML via the [xerces plugin](https://www.libelektra.org/plugins/xerces). What is your intended use case?
